web/src/components/MapView.tsx

```diff
@@ -14,9 +15,12 @@ import {
   mapMarker
 } from 'api-client'
 
-import MapPopup from './MapPopup'
+import { getSnapPoints } from '../utils/getSnapPoints'
 
+// TODO refactor height
```

Comment:
i couldn't find a way to set the height: 100%. Some help required

Reply:
Not quite sure, what you mean right here.

Reply:
Same for me, seems like the map has 100% height for me.

Reply:
you can try to set here `height: 100%` and check that nothing gets rendered i dunno how to set the height without calculating it here

Reply:
I think the comments are more about that the comment is not really explanatory and adds more confusion than it solves.
```suggestion
// Workaround since nothing is rendered if height is set to 100% 
```